docs/remote_svs.rst

```diff
@@ -0,0 +1,189 @@
+Accessing SecureDrop Remotely
+=============================
+
+While it's necessary for SecureDrop servers to be hosted on-premise within your
+organization, and for administrators to retain direct physical access in order
+to perform any necessary maintenance or troubleshooting tasks, it is possible
+for journalists to use SecureDrop in remote locations.
+
+Risk Mitigation for Remote Secure Viewing Stations
+--------------------------------------------------
+
+One option to allow uninterrupted access to SecureDrop is to set up a remote
+Secure Viewing Station for use by individual journalists.
```

Comment:
This chapter uses a few glossary terms like *Secure Viewing Station*; per our style guide let's italicize those.